### DISCUSSION TITLE
GA4 ne passe pas par défaut avec GTM

### DISCUSSION THREAD

**Discussion by fmedard** [AmauriC/tarteaucitron.js#943] [Q&A]
Bonjour,
Je fais passer mes propriétés GA3 et GA4 via GTM, avec Tarteaucitron en mode automatique.
Le mode automatique de Tarteaucitron détecte et propose uniquement GTM à l'internaute. 
- Si l'internaute refuse tous les services (donc uniquement GTM) le fonctionnement est normal, aucun cookie ne passe.
- Si l'internaute accepte tous les services, GA3 passe bien dans les Analytics, mais pas GA4.
- Le container GTM a bien été testé en mode preview et fonctionne normalement 
- Si je retire l'appel au script Tarteaucitron, GA3 et GA4 fonctionnent.

Après investigation, en comparant les cookies posés avec et sans le script Tarteaucitron, je constate qu'un cookie manque à l'appel avec Tarteaucitron quand j'autorise tous les services : _ga_EVP8... dans la copie d'écran jointe.
<img width="370" alt="image" src="https://user-images.githubusercontent.com/29124974/169080069-340c752e-036f-488c-bdad-370e9fda6463.png">

J'ai réussi à "régler le problème" en passant ma configuration Tarteaucitron en manuel, avec les services GTM, GA3 et GA4.

Avez-vous rencontré ce problème, et est-il possible de faire fonctionner GTM avec GA4 en mode automatique ? Bizarrement, GA3 fonctionne par défaut

**Comment by AmauriC** [comment]
Répondu sur https://github.com/AmauriC/tarteaucitron.js/issues/948 ; )
